tests/mr/device/mr_multithreaded_tests.cpp

```diff
@@ -191,6 +192,7 @@ void allocate_loop(rmm::mr::device_memory_resource* mr,
     void* ptr        = mr->allocate(size, stream);
     {
       std::lock_guard<std::mutex> lock(mtx);
+      EXPECT_EQ(cudaSuccess, cudaEventRecord(event, stream.value()));
```

Comment:
Just use RMM_CUDA_TRY -- it will throw on failure which will cause the test to fail.

```suggestion
      RMM_CUDA_TRY(cudaEventRecord(event, stream.value()));
```